docs/ENVVARS.md

```diff
@@ -0,0 +1,38 @@
+# Environment Variables
+
+The follow describes environment variables available to be set when running Tink Server or Tink CLI.
+
+| Name                                                                                           | Type   | Service(s)        | Description                                                                                                                                   |
+| ---------------------------------------------------------------------------------------------- | ------ | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `TINK_AUTH_USERNAME=tink`                                                                      | string | server            | username to use for basic auth to http endpoints                                                                                              |
+| `TINK_AUTH_PASSWORD=tink`                                                                      | string | server            | password to use for basic auth to http endpoints                                                                                              |
+| `TINKERBELL_CERT_URL=http://127.0.0.1:42114/cert`                                              | string | cli/worker        | string url from which to get a TLS certificate                                                                                                |
```

Comment:
should Type say `string/url` and avoid saying so in Description? Also TLS certificate for what? Its probably a good idea to be explicit since tls connections usually just work and don't need this kind of feeding into a binary.